### DISCUSSION TITLE
Your server does not serve wasm with `application/wasm` MIME type

### DISCUSSION THREAD

**Discussion by jimepler** [ruffle-rs/ruffle#7480] [Q&A] [answered]
I added ruffle to my website and added the wasm mime type in my cpanel MIME type section. Also in .htaccess. 

I still see the following message in the Chrome console that my server isn't serving it properly.  

WebAssembly.instantiateStreaming` failed because your server does not serve wasm with `application/wasm` MIME type. Falling back to `WebAssembly.instantiate` which is slower. Original error:
 TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
 
 Do I need to contact my hosting company to configure anything on their end?

**Comment by danielhjacobs** [comment, marked as answer]
See https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-webassembly-mime-type

**Reply by jimepler** [reply]
I guess you didn't interpret my question correctly. I did what was instructed on that page but am still getting the error I noted in my original post.

**Reply by jimepler** [reply]
My hosting company is running CloudLinux so many I need to contact them directly to do something. Just adding the mime type in cPanel doesn't seem to be enough

**Reply by jimepler** [reply]
For the record, it appears CloudLinux runs Apache mod_lsapi PRO which in turn is based on LiteSpeed, so I'm going to have the tech support use the LiteSpeed steps in the documentation on that page.
